docs/guide/upgrading.md

```diff
@@ -56,25 +57,27 @@ Your project specific modules should not use the `@apostrophecms` namespace to a
   - Module field schemas now use an object structure on the `fields` property. It has `add`, `remove`, and `group` subproperties to replace A2's `addFields`, `removeFields`, and `arrangeFields`.
   - `columns` (for piece manager UI columns) is structured similarly to `fields` with `add`, `remove`, and `order` subproperties. This replaces A2's `addColumns` property.
   - `columns` (for piece manager UI columns) is structured similarly to `fields` with `add`, and `remove` subproperties. This replaces A2's `addFilters` property.
-  - Many module options that were top level properties in A2 (e.g., `label`, `alias`, `sort`) are now subproperties of `options`. These include all options with primitive value (e.g., booleans, string, numbers) in addition to select object or array options. See the [module options reference](/reference/module-api/module-options.md).
+  - Many module options that were top level properties in A2 (e.g., `label`, `alias`, `sort`) are now subproperties of `options`. These include all settings that are *not* covered in the [module properties overview reference](/reference/module-api/module-overview.md). See the [module options reference](/reference/module-api/module-options.md) for more on that.
```

Comment:
How about "Many module settings". In A3, strictly speaking, if it ain't in `options`, it ain't an option, it's something else.